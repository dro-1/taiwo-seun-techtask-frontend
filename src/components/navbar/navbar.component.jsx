import "./navbar.styles.scss";

export const Navbar = () => (
  <header>
    <h1>bl1p</h1>
  </header>
);
