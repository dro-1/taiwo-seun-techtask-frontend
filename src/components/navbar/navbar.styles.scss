header {
  background-color: rgb(7, 7, 58);
  padding: 1rem;
  h1 {
    text-transform: uppercase;
    color: white;
    font-size: 3rem;
    text-align: center;
  }
}
