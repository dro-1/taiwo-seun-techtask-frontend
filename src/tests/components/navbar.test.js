import React from "react";
import { shallow } from "enzyme";
import { Navbar } from "../../components/navbar/navbar.component";

describe("Navbar Component", () => {
  it("expect to render Navbar", () => {
    const navbar = shallow(<Navbar />);
    expect(navbar).toMatchSnapshot();
  });
});
