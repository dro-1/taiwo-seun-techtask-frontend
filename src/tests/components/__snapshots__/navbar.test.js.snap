// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component expect to render Navbar 1`] = `ShallowWrapper {}`;
